### PR TITLE
Add Upcount

### PR DESCRIPTION
**Pull request by madisvain**
[Upcount.app](https://www.upcount.app/) is an Invoicing application focusing on ease of use and offline-first approach. The application is fully open source & uses Electron - has an automated Github actions build process and is looking for [ideas, suggestions](https://github.com/madisvain/upcount/issues) or contributors.

**By submitting this pull request, I promise I have read the [contributing guidelines](https://github.com/sindresorhus/awesome-electron/blob/master/contributing.md) twice and ensured my submission follows it. I realize not doing so wastes the maintainers' time that they could have spent making the world better. 🖖**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -186,6 +186,7 @@ Made with Electron.
 - [Motrix](https://github.com/agalwood/Motrix) - Download manager.
 - [Franz](https://github.com/meetfranz/franz) - Skype, Slack, Hangouts, WhatsApp, Grape, Telegram, FB Messenger, Hipchat in the same app.
 - [Gmail Desktop](https://github.com/timche/gmail-desktop) - Unofficial Gmail app.
+- [Upcount](https://github.com/madisvain/upcount) - Invoice with ease. Open source, configurable & offline first.
```
Comment (sindresorhus) [inline readme.md]: "Open source" is implied by it being in this list. I prefer a very short description of what the app is.

Comment (madisvain) [reply readme.md]: @sindresorhus understood and made the description super simple "Easy Invocing app"

File: readme.md [hunk 2]
```diff
@@ -186,6 +186,7 @@ Made with Electron.
 - [Motrix](https://github.com/agalwood/Motrix) - Download manager.
 - [Franz](https://github.com/meetfranz/franz) - Skype, Slack, Hangouts, WhatsApp, Grape, Telegram, FB Messenger, Hipchat in the same app.
 - [Gmail Desktop](https://github.com/timche/gmail-desktop) - Unofficial Gmail app.
+- [Upcount](https://github.com/madisvain/upcount) - Easy Invocing app
```
Comment (sindresorhus) [inline readme.md]: You have a typo and you forgot the `.`

Comment (madisvain) [reply readme.md]: @sindresorhus I'm very sorry about these mistakes and taking your time. I hope this final commit will solve the remaining issues.

**Pull request merged**